src/ForkCMS/Bundle/InstallerBundle/Service/RequirementsChecker.php

```diff
@@ -278,13 +278,21 @@ protected function checkAvailableFiles()
     /**
      * Ensure that Apache .htaccess file is written and mod_rewrite does its job
      */
-    protected function checkApacheRewrites()
+    protected function checkServerRewrites()
     {
-        $this->checkRequirement(
-            'modRewrite',
-            (bool) (getenv('MOD_REWRITE') || getenv('REDIRECT_MOD_REWRITE')),
-            self::STATUS_WARNING
-        );
+        if (!preg_match('/^nginx/i', $_SERVER['SERVER_SOFTWARE'])) {
```

Comment:
But do you agree that the condition should be `stripos($_SERVER['SERVER_SOFTWARE'], 'nginx') === false` instead of `stripos($_SERVER['SERVER_SOFTWARE'], 'nginx') === 0`